content/docs/start/experiments.md

```diff
@@ -221,3 +221,7 @@ All these commands also accept
 [Git revisions](https://git-scm.com/docs/gitrevisions) (commits, tags, branch
 names) to compare. This is a powerful mechanism for navigating experiments to
 see the history, to pick the best ones, etc.
+
+If you report more than one metric to the `plots` file, you can provide 
+metric names to `dvc plots diff` command with `-x` and `-y` options.
+See more here: [`dvc plots diff` options](https://dvc.org/doc/command-reference/plots/diff#options).
```

Comment:
Probably not needed. WDYT @shcheklein ?

If keeping this, I'd put it in a block quote (`>`) and start "The -x and -y options select which data series..." or something like that but it's going to be difficult to write it in a way that doesn't require further explanations (for example what is a data series?) so again, probably best to avoid this...

Reply:
Maybe just "> See `dvc plots diff` for more info on its options, such as `-x` and `-y` used above."

Reply:
Notice that there's no need to make manual links from DVC commands. they get auto-linked.

Reply:
The goal of this paragraph, when I wrote this, was to: 

1) Explain how to plot different metrics if needed, and explain that that's actually possible, spark interest to experiment with different metrics like that.
2) Give an example of how to do this (actually, the example plot above does this in an implicit way).

You guys probably have a better judgement on whether it's a good idea to explain this in the get started series.

Reply:
I think we want to keep the GS as simple and focused as possible. Users with interest on this can click on the `dvc plots` link that already exists above and learn everything about metrics and plots. We also don't talk about `dvc plots show`, for example... No way to cover all these options in GS directly. I personally vote to remove this note altogether but I'm OK with a very short one like I suggested above, since this is the very bottom of the doc anyway.

Reply:
Yep, I would omit it as well. In other places if explain options we explain them in the `<details>` section. Otherwise PR looks great to me!